packages/stems/src/components/SegmentedControl/types.ts

```diff
@@ -39,4 +39,9 @@ export type SegmentedControlProps<T extends string> = {
    */
   label?: string
   'aria-labelledby'?: string
+
+  /**
+   * A hack to allow shifting animations to settle before recalculating the tab width
+   */
+  forceRefreshAfterMs?: number
```

Comment:
just the hack the doctor ordered, lmao. lets go with this for now 100% lol

Reply:
😬